python/py-torchaudio/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        pytorch audio 0.8.1 v
+revision            0
+name                py-torchaudio
+
+categories-append   audio
+platforms           darwin
+supported_archs     noarch
+license             BSD-2
```

Comment:
Can you fix the lint error here regarding the license type, see above.

port lint --nitpick py38-torchaudio:
Error: Invalid license 'BSD-2': use BSD instead

Reply:
Thanks done. Also removed the `noarch` as this compiles against `libsox`.